src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -1,4 +1,36 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "USERS")
+@AllArgsConstructor
+@NoArgsConstructor
 public class User {
+    @Id
+    @GeneratedValue
+    private Long id;
```

Comment:
Wszędzie, to i tu się czepnę :D

@Id
@GeneratedValue(strategy = GenerationType.IDENTITY)

- bez takiej adnotacji  w mojej encji miałem problem z ręcznym dodaniem rekordu przez Workbench'a. Stanęło na tym że wszędzie tak robimy

Oraz nazwy ID - ustalmy czy w każdej encji mamy Long id, czy Long userId, groupId, CarrtId itp (To myślę że nie tylko estetyka, ale może ułatwić późniejsze wysyłanie zapytań SQL - nie trzeba będzie pamiętać czy w danej encji jest samo id czy np. userId)